### ISSUE TITLE
Update stable version in repo from 0.0.0 to 0.2.0

### ISSUE THREAD

**Issue by GermanBluefox** [MyHomeMyData/ioBroker.flexcharts#55]
# Think about adding version 0.2.0 to stable repository.
**Version**: stable=**0.0.0** (0 days old) => latest=**0.2.0** (30 days old)
**Installs**: stable=**0** (0%), latest=**26** (74.29%), total=**35**

Click to use [developer portal](https://www.iobroker.dev/adapter/flexcharts/releases)
Click to [edit](https://github.com/ioBroker/ioBroker.repositories/edit/master/sources-dist-stable.json)

**Do not close this issue manually as a new issue will be created if condition for update still exists.**

Please drop a comment if any reason exists which blocks updating to version 0.2.0 at this time.

Note: This is an automatically generated message and not personally authored by bluefox!
      @mcm1957 for evidence

**Comment by hombach** [comment]
YES PLEASE!!!! This Adapter is worth to be visible in stable!!!!  
@MyHomeMyData : perhaps this might be of interest for you: https://forum.iobroker.net/topic/67668/tibberlink-adapter/489

**Comment by MyHomeMyData** [comment]
@hombach Thanks for your feedback! I'll go ahead within the next few days.

**Comment by MyHomeMyData** [comment]
flexcharts version 0.3.0 is available in stable repo.
